packages/dusk-cli/src/lib/extractMultiplayerMetadata.ts

```diff
@@ -15,6 +16,9 @@ export function extractMultiplayerMetadata(logicJsContent: string) {
   const maxPlayersString = logicJsContent.match(regexes.maxPlayers)?.at(1)
   const maxPlayers = maxPlayersString ? parseInt(maxPlayersString) : undefined
 
+  const landscapeString = logicJsContent.match(regexes.landscape)?.at(1)
+  const landscape = landscapeString ? landscapeString === "true" : undefined
```

Comment:
Couldn't this just be `const landscape = landscapeString === "true"`? Or is it important that it be `undefined` and not `false`?

Reply:
It's just for consistency. You can see that all other fields are defined as optional in the resulting type. We could make them required, but we should make all of them required in that case 🙂

Reply:
makes sense to me!